### DISCUSSION TITLE
Facebook投稿のサムネイルがエラーになっている

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#491] [要望・アイデア・不具合]
https://www.facebook.com/qiita の投稿で"Bitly | Too Many Requests | 429 Error"となっているものが最近増えているように思います。

<img width="713" alt="Screenshot 2023-06-14 at 11 44 39" src="https://github.com/increments/qiita-discussions/assets/1148320/b80e42bf-6409-47a2-8d8b-f8b0440deb70">

一瞬「これは何のリンクだろう？」と困惑するので原因を調査してもらえるとありがたいです。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご報告ありがとうございます！🙏
調査して対応します！

**Comment by wataru86** [comment]
@JunichiIto
調査したところ、Facebook が Bitly に OGPを取得するタイミングによっては `Too Many Requests` のエラーになってしまう場合があり、それによりエラーページのデータがキャッシュされている様子でした。エラーになっている投稿のキャッシュ削除も試みましたが、Facebookで操作した際にエラーが発生し、まだ修正できていない状況です。

本日Qiitaアカウントから投稿された分に関しては同様の事象が発生していないので、しばらく様子見したいと思います。それでもまた頻発する場合は Bitly 以外の短縮URL発行サービスの検討を行いたいと思います。

**Reply by JunichiIto** [reply]
ご対応ありがとうございます。了解しました！

**Reply by wataru86** [reply]
10日ほど様子を見ましたが現在も同様の事象は発生していないため、こちらの discussion は一旦 close とさせていただきます！
